packages/blueprints/blueprint/src/blueprint.ts

```diff
@@ -318,3 +315,74 @@ function structureExistingBlueprints(location: string | undefined): BlueprintIns
   }
   return [];
 }
+
+/**
+ * This generates the default PR information that resynth updates will go onto.
+ */
+function defaultResynthPR(params: { options: any; context: Context }): {
+  branch: string;
+  title: string;
+  description: string;
+} {
+  if (params.context.project.blueprint.instantiationId) {
+    // The blueprint options/versions are being changed.
+    return {
+      branch: `update-blueprint-${formatBlueprintName(
+        params.context.package.name || 'unknown',
+      )}-${params.context.project.blueprint.instantiationId.slice(0, 10)}`,
```

Comment:
nit: this slices a UUID in a kinda awkward position:
```
> "8f859105-3ca2-4865-b3f1-b0dc2fa687ce".slice(0, 10)
'8f859105-3'
```

maybe hash the ID then slice, like you do for the time

Reply:
or slice in a different position, since it's kinda nice for the instantiation ID to be human readable

Reply:
Yeah changing this to 8 seems better